flaml/onlineml/autovw.py

```diff
@@ -0,0 +1,157 @@
+                 init_config: dict,
+                 search_space: dict,
+                 max_live_model_num: int,
+                 min_resource_lease: Optional[float] = 'auto',
+                 automl_runner_args: Optional[dict] = {},
+                 scheduler_args: Optional[dict] = {},
+                 model_select_policy: Optional[str] = 'threshold_loss_ucb',
+                 metric: Optional[str] = 'mae_clipped',
+                 random_seed: Optional[int] = None,
+                 model_selection_mode: Optional[str] = 'min',
+                 cb_coef: Optional[float] = None,
+                 ):
+        """Constructor
+
+        Args:
+            init_config: A dictionary of a partial or full initial config,
+                e.g. {'interactions': set(), 'learning_rate': 0.5}
+            search_space: A dictionary of the search space. This search space includes both
+                hyperparameters we want to tune and fixed hyperparameters. In the latter case,
+                the value is a fixed value.
+            max_live_model_num: The maximum number of 'live' models, which, in other words,
+                is the maximum number of models allowed to update in each learning iteraction.
+            min_resource_lease: The minimum resource lease assigned to a particular model/trial.
+                If set as 'auto', it will be calculated automatically.
+            automl_runner_args: A dictionary of configuration for the OnlineTrialRunner.
+                If set {}, default values will be used, which is equivalent to using the following configs.
+                automl_runner_args =
+                    {"champion_test_policy": 'loss_ucb' # specifcies how to do the statistic test for a better champion
+                    "remove_worse": False              # specifcies whether to do worse than test
+                    }
+            scheduler_args: A dictionary of configuration for the scheduler.
+                If set {}, default values will be used, which is equivalent to using the following configs.
+                scheduler_args =
+                    {"keep_challenger_metric": 'ucb' # what metric to use when deciding the top performing challengers
+                    "keep_challenger_ratio": 0.5     # denotes the ratio of top performing challengers to keep live
+                    "keep_champion": True            # specifcies whether to keep the champion always running
+                    }
+            model_select_policy: A string to specify how to select one model to do prediction
+                from the live model pool
+            metric: A string to specify the name of the loss function used for calculating
+                the progressive validation loss
+            random_seed (int): An integer of the random seed used in the searcher
+                (more specifically this the random seed for ConfigOracle)
+            model_selection_mode: A string in ['min', 'max'] to specify the objective as
```

Comment:
does it make sense for `model_select_policy` and `metric` here to also add the available strings like we do in `model_selection_mode`?